src/t8_cmesh/t8_cmesh_commit.cxx

```diff
@@ -234,7 +234,7 @@ t8_cmesh_commit_partitioned_new (t8_cmesh_t cmesh, sc_MPI_Comm comm)
   sc_flops_snap (&fi, &snapshot);
 #endif
 
-  T8_ASSERT (t8_cmesh_comm_is_valid (cmesh, comm));
+  T8_ASSERT (t8_cmesh_get_mpicomm (cmesh) == comm);
```

Comment:
I would still use `t8_cmesh_comm_is_valid ` here because of the straighforward naming
```suggestion
  T8_ASSERT (t8_cmesh_comm_is_valid (cmesh, comm);
```